transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/ListTransferMethodActivity.java

```diff
@@ -129,9 +130,14 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 
     @Override
     public void showSelectTransferMethodView() {
-        Intent myIntent = new Intent(ListTransferMethodActivity.this, SelectTransferMethodActivity.class);
-        myIntent.addFlags(Intent.FLAG_ACTIVITY_SINGLE_TOP);
-        startActivityForResult(myIntent, SELECT_TRANSFER_METHOD_REQUEST_CODE);
+        Intent intent = new Intent();
+        intent.setAction(HyperwalletIntent.ACTION_SELECT_TRANSFER_METHOD);
+        if (intent.resolveActivity(getPackageManager()) != null) {
```

Comment:
Oh I thought you want it to be like that because its described in the ticket HW-55513, I did not comment back because I thought the module will be broken to smaller module?, iff we won't break this in the future, and I agree we should use explicit intents for same modules